lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -2,30 +2,26 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
-import { AuthError, Account, AuthResponse, AuthorizationCodeModule, AuthenticationParameters, INetworkModule, TokenResponse, UrlString, TemporaryCacheKeys, TokenRenewParameters } from "msal-common";
-import { BrowserStorage } from "../cache/BrowserStorage";
+// Common package imports
+import { Account, AuthorizationCodeModule, AuthenticationParameters, INetworkModule, TokenResponse, UrlString, TemporaryCacheKeys, TokenRenewParameters } from "msal-common";
```

Comment:
These references need to be updated to `@azure/msal-common`

Reply:
done.